audiomentations/augmentations/transforms.py

```diff
@@ -193,32 +193,69 @@ class AddGaussianSNR(BaseWaveformTransform):
 
     supports_multichannel = True
 
-    def __init__(self, min_SNR=0.001, max_SNR=1.0, p=0.5):
+    def __init__(self, 
+                 min_snr_in_db=3,
```

Comment:
Some people may use this class like `AddGaussianSNR(0.01, 0.001)`. With this new parameter ordering, those numbers would be interpreted as db although they are not. Perhaps it's better to swap the order of parameters, so it doesn't become a breaking change (for now)?

i.e.
```
min_SNR=0.001,
max_SNR=1.0
min_snr_in_db=None,
max_snr_in_db=None,
```

(but people should still get the warning if they use the old parameters)

Reply:
I've fixed that.

Reply:
Another alternative would be to enforce keyword-only arguments (instead of allowing both positional and keyword arguments). But if we go down that path, we would maybe have to do the same in all other transforms as well for consistency, so I don't think that's the best solution

Reply:
I think that change of arguments order is the best choice for now.